libraries/wasm-jit/Source/WAST/NFA.cpp

```diff
@@ -514,7 +514,7 @@ namespace NFA
 		std::set<StateIndex> terminalStates;
 
 		CharSet* classCharSets = (CharSet*)alloca(sizeof(CharSet) * numClasses);
-		memset(classCharSets,0,sizeof(CharSet) * numClasses);
+    memset((char*)classCharSets,0,sizeof(CharSet) * numClasses);
```

Comment:
Does this just mask the warning? It's still memseting a non trivial type.

A `CharSet` is a `DenseStaticIntSet<U8,256>` and `DenseStaticIntSet` actually does do a memset on its contents. Since this isn't performance sensitive I wonder if just replacing this with a std::vector or such makes sense

Reply:
`dumpDFAGraphViz()` is never called unless `-D_DEBUG` is passed, which it is not.

Reply:
good call. just zap it all then

Reply:
Did you mean `_DEBUG` flag would never be used and I can go ahead removing all `#ifdef _DEBUG` and the functions which were only used inside `#ifdef _DEBUG` block like `dumpDFAGraphViz`?

Reply:
Right, `_DEBUG` will never be set, so might as well just remove this code since it's not too invasive of change.